manifests/params.pp

```diff
@@ -35,6 +35,10 @@
   # Pulp max speed setting
   $pulp_max_speed = undef
 
+  # Qpid perf settings
+  $qpid_session_unacked = 10
```

Comment:
The qpid in EPEL doesn't accept this parameter and failing to start up. I think we should revert this.